maze_transformer/generation/latticemaze.py

```diff
@@ -329,15 +334,18 @@ def from_adjlist(
         )
 
     @classmethod
-    def from_tokens(cls, maze_tokens: list[str]) -> "LatticeMaze":
+    def from_tokens(cls, tokens: list[str]) -> "LatticeMaze":
         """create a LatticeMaze from a list of tokens"""
-
+        adjlist_tokens = get_adjlist_tokens(tokens)
         edges: list[str] = list_split(
```

Comment:
Is this well behaved if one passes in `["(0,0)", ...., "(3,4)"]` without the start/end tokens?

I ask because when I extract the adjacency list from a tokenized sequence I typically throw away the enclosing special tokens ; would be nice if most functions weren't sensitive to their presence.